src/views/PeopleSearch/components/PeopleSearchResult/index.js

```diff
@@ -188,34 +204,37 @@ export default class PeopleSearchResult extends Component {
                 padding: '1rem',
               }}
             >
-              <Grid item xs={1}>
-                <IMG
-                  className="people-search-avatar"
-                  src={`data:image/jpg;base64,${this.state.avatar}`}
-                  alt=""
-                  noLazyLoad="true"
-                  placeholderColor="#eeeeee"
-                />
+              <Grid item xs={5}>
+                <div style={{ display: 'flex', alignItems: 'center' }}>
+                  <IMG
+                    className="people-search-avatar"
+                    src={`data:image/jpg;base64,${this.state.avatar}`}
+                    alt=""
+                    noLazyLoad="true"
+                    placeholderColor="#eeeeee"
+                  />
+                  <div>
+                    <Typography>
+                      {Person.FirstName} {nickname} {Person.LastName}
+                    </Typography>
+                    <Typography style={{ fontSize: '85%' }}>
+                      {Person.Email?.includes('.') ? Person.Email : null}
+                    </Typography>
+                  </div>
+                </div>
```

Comment:
You can use the pre-existing Grid to align these, instead of adding an extra `div`.
```suggestion
              <Grid item xs={5} container alignItems="center">
                  <IMG
                    className="people-search-avatar"
                    src={`data:image/jpg;base64,${this.state.avatar}`}
                    alt=""
                    noLazyLoad="true"
                    placeholderColor="#eeeeee"
                  />
                  <div>
                    <Typography>
                      {Person.FirstName} {nickname} {Person.LastName}
                    </Typography>
                    <Typography style={{ fontSize: '85%' }}>
                      {Person.Email?.includes('.') ? Person.Email : null}
                    </Typography>
                  </div>
```

Reply:
Done.